### PR TITLE
RAAP-388 Added Jenkinsfile

### PR DESCRIPTION
**Pull request by cp2boston**

### CODE REVIEW

**Comment by seth-mg** [comment]
@cp2boston if you get a chance can add a `-B` to all the `mvn` calls in `examples/docker/run_java.sh`?

File: Jenkinsfile.examples [hunk 1]
```diff
@@ -0,0 +1,36 @@
+node {
+    def SOURCEDIR = pwd()
+    def TEST_CONTAINER = 'examples/java-test'
+    def DOCKERFILE_DIR = './examples/docker'
+    def PUBLISHED_VERSION = '1.7.3'
```
Comment (seth-mg) [inline Jenkinsfile.examples:5]: If there's release instructions somewhere, can we get them updated to ensure that this gets bumped too?

**Comment by seth-mg** [comment]
And it looks like master needs to be merged into the branch before it can be delivered.

File: Jenkinsfile.examples (hunk 1 above)
Comment (cp2boston) [reply Jenkinsfile.examples:5]: Good idea.  I think that it can be added to the binding publish script, so that it's automatic.

**Review by seth-mg — APPROVED** [review]

**Pull request merged**